authentication/src/main/java/io/camunda/authentication/entity/CamundaUser.java

```diff
@@ -54,15 +48,17 @@ public CamundaUser(
       final String displayName,
       final String username,
       final String password,
-      final List<String> roles,
+      final List<? extends GrantedAuthority> authorities,
```

Comment:
@Ben-Sheppard @maryarm Here I have separated between Roles and spring's authority concept. can you confirm that the spring's `GrantedAuthority` concept will not be used with `auth-basic` profile? and will maybe kept for the default `auth` profile, that is used by Operate and Tasklist webapps?

Reply:
Separating these IMO is fine, the old `auth` profile will not be kept so this field likely won't be used